src/main/java/com/teragrep/pth_06/planner/HdfsQueryProcessor.java

```diff
@@ -93,7 +103,7 @@ public HdfsQueryProcessor(Config config) {
                 );
             }
         }
-        if (topicsRegexString == null) {
+        if (topicsRegexString.isEmpty()) {
             topicsRegexString = "^.*$"; // all topics if none given
```

Comment:
I think `topicRegexString` could be initialized with this, as it would get overwritten anyways

Reply:
Solved in commit cc54c576ec45b69d66b5e2bbc41adf5570772610